app/assets/javascripts-es6/item-select.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(() => {
+  const items = document.getElementsByClassName("item");
+  for(let i=0; i<items.length; i++){
+    items[i].addEventListener("click", onItemSelect);
+  }
```

Comment:
`items.forEach(item => items.addEventListener("click", onItemSelect));`

Generic `for` loops are pretty uncommon in ES6 code. Generally you can find a way to do it using one of the `Array.prototype` methods.

Reply:
Gotcha. I just tried a `forEach` but it didn't work. I believe the reason is that what `getElementsByClassName` returns is not strictly speaking an array but a HTML collection. However this works- ` [].forEach.call(items, item => item.addEventListener("click", onItemSelect));
`